### PR TITLE
Fix styling for secondary button

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/components/Button/index.tsx [hunk 1]
```diff
@@ -7,15 +7,14 @@ export type ButtonVariant = (
...
-    | 'inverted'
     | 'action'
 );
 
 const buttonVariantToStyleMap: { [key in ButtonVariant]: string; } = {
     primary: styles.primary,
     secondary: styles.secondary,
     tertiary: styles.tertiary,
-    inverted: styles.inverted,
+    // inverted: styles.inverted,
```
Comment (AdityaKhatri) [inline src/components/Button/index.tsx]: Remove from here as well

**Comment by AdityaKhatri** [comment]
Plz typecheck fix sir plz

File: src/hooks/useParentPositionTracking.ts [hunk 2]
```diff
@@ -20,6 +20,8 @@ function useParentPositionTracking(
             const { current: el } = elementRef;
             if (el && el.parentElement) {
                 const elementBCR = el.parentElement.getBoundingClientRect();
+                const s = getComputedStyle(el.parentElement);
+                console.info(s.getPropertyValue('border-width'), elementBCR.width);
```
Comment (tnagorra) [inline src/hooks/useParentPositionTracking.ts]: Do we need this console.info

**Review by tnagorra — APPROVED** [review]

File: src/hooks/useParentPositionTracking.ts (hunk 2 above)
Comment (frozenhelium) [reply src/hooks/useParentPositionTracking.ts]: naah

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**